### PR TITLE
add datasteward circle

### PR DESCRIPTION
**Pull request by Brilator**
- would need more detail
- and needs to be shaped into a recurrent event once the mechanism is implemented 

@fstein88

### CODE REVIEW

File: src/content/events/2024-10-16_DataSteward-Circle.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
+---
+title: DataSteward Circle
+date: 2024-10-11
+category: Meeting
+audience: [DataStewards]
+excerpt: 'Exchange best practices about creating ARCs. Updates on services, templates and tools, that will hopefully make your work as Data Stewards easier'
```
Comment (fstein88) [inline src/content/events/2024-10-16_DataSteward-Circle.md:6]: #
```suggestion
excerpt: 'Exchange best practices about creating ARCs. Updates on tools and services as well as annotation support, that will make your work as Data Stewards easier.'
```

File: src/content/events/2024-10-16_DataSteward-Circle.md [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+start: 2024-10-16T09:00:00
+end: 2024-10-16T10:00:00
+mode: Online
+image: ~/assets/images/DataPLANT_logo_square_bg_darkblue.svg
+location:
+  short: Teams
```
Comment (fstein88) [inline src/content/events/2024-10-16_DataSteward-Circle.md:12]:
```suggestion
  short: Microsoft Teams
```

File: src/content/events/2024-10-16_DataSteward-Circle.md
Comment (fstein88) [inline src/content/events/2024-10-16_DataSteward-Circle.md:1]: Add:

## Support chat

Feel free to join the chat via our Element room: [#datastewardexchange:matrix.uni-freiburg.de](https://matrix.to/#/!TeCdVdqWpYPpzZNTQs:matrix.uni-freiburg.de?via=matrix.uni-freiburg.de&via=matrix.org). Directly contact us there for data stewardship, to raise questions, and to discuss topics for upcoming meetings with us.

File: src/content/events/2024-10-16_DataSteward-Circle.md [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+  url: https://teams.microsoft.com/l/meetup-join/19%3ameeting_MWU2NWYwNTYtYmY3OS00OWZkLThkM2YtNWQzMWJlODAwZWZl%40thread.v2/0?context=%7b%22Tid%22%3a%229071867c-98f0-4006-89aa-4e4fd55af39d%22%2c%22Oid%22%3a%221c9911a0-ac64-4f61-8904-9ec3eeb64f98%22%7d
+registration: 
+  description: Please register via email. You’ll receive a reminder with a short agenda shortly before each meeting.
+  url: mailto:info@nfdi4plants.org?subject=Sign%20up%20for%20DataSteward%20Circle%20mailing%20list&body=Dear%20DataPLANT%2C%20%0A%0Aplease%20sign%20me%20up%20for%20the%20DataSteward%20Cirlce%20mailing%20list.%20
+---
+
+## Topics
+
+There will be an update on new developments in tools and services (~ 10 min) as well as regarding templates and terminology (~ 10 min). For the rest of the time, we will have a look at ARCs and how research data is documented in them. One person will present their current work and how they went about in structuring the ARC.
```
Comment (fstein88) [inline src/content/events/2024-10-16_DataSteward-Circle.md:21]: Alternative for continuous text:
- ARCification: How to document research data in an ARC
- Technology: Short updates on new developments in tools and services  
- Annotation support: Short updates on template and terminology development

**Review by fstein88 — COMMENTED** [review]
Added/edited a few points

**Review by Freymaurer — APPROVED** [review]

**Pull request merged**